src/anemoi/training/train/forecaster.py

```diff
@@ -290,6 +290,17 @@ def get_feature_weights(
 
         return torch.from_numpy(loss_scaling)
 
+    @staticmethod
+    def get_node_weights(config: DictConfig, graph_data: HeteroData) -> torch.Tensor:
+        node_weights = graph_data[config.graph.data][config.model.node_loss_weight].squeeze()
+
+        if "spatial" in config.training.loss_scaling:
+            spatial_loss_scaler = instantiate(config.training.loss_scaling.spatial)
+            node_weights = spatial_loss_scaler.area_weights(graph_data)
+            LOGGER.info("Rescaling area weights")
+
+        return node_weights
+
```

Comment:
As we discussed, I like the idea of a way to scale/modify the node weights. 
So I think that this function can be config controlled, and allow arbitary modifications to be applied.